_shows/20_21/autumn_showers.md

```diff
@@ -0,0 +1,59 @@
+---
+# Check out the Show Docs for more information
+# -- https://history.newtheatre.org.uk/docs/show/
+# Remove or comment any lines that are not being used
+
+title: Autumn Showers
+playwright: Alexandra Hogg
+student_written: true
+period: Autumn
+season: Online
+season_sort: 390
+date_start: 2020-11-29
+date_end: 2020-11-29
+
+cast:
+- role: Sofia 1
+  name: Katie Booth
+- role: Sofia 2
+  name: Lily Bailes
+- role: Katherine
+  name: Megan Peace
+
+crew: 
+- role: Director
+  name: Alexandra Hogg
+- role: Producer
+  name: Charlie Basley
+- role: Assistant Producer
+  name: Lottie Wright
+- role: Shadow Director
+  name:Lauren Brady
+- role: Video Editor
+  name: Lauren Brady
+- role: Shadow Producer 
+  name: Video Editor
+  name: Alicia Conte
+- role: Sound Designer
+  name: Charlie Finney
+- role: Video Editor
+  name: Charlie Finney
+- role: Poster Designer 
+  name: Ace Edwards
+- role: Captioner
```

Comment:
We should pick whether the role is "Captioner" (as here) or "Caption Writer" (as above), unless these are somehow different roles, for consistency.

Reply:
I am happy with whichever one

Reply:
Shall we decide as 'Captioner' for consitency?